xml/admin_gui_dashboard.xml

```diff
@@ -1183,673 +1317,771 @@
+   <guimenu>Targets</guimenu> table and click <guimenu>Edit</guimenu> in the
+   top left of the table.
+  </para>
+  <para>
+   You can then modify the &iscsi; target, add or delete portals, and add or
+   delete related RBD images. You can also adjust authentication information
+   for the gateway.
+  </para>
+ </sect3>
+ <sect3 xml:id="dashboard-iscsi-delete">
+  <title>Deleting &iscsi; Targets</title>
+  <para>
+   To delete an &iscsi; target, select the table row and click the drop-down arrow
+   next to the <guimenu>Edit</guimenu> button and select
+   <guimenu>Delete</guimenu>. Activate <guimenu>Yes, I am sure</guimenu> and
+   confirm with <guimenu>Delete iSCSI target</guimenu>.
+  </para>
+ </sect3>
+</sect2>
+
+<sect2 xml:id="dash-rbd-qos">
+ <title>RBD Quality of Service (QoS)</title>
+ <tip>
+  <title>For More Information</title>
+  <para>
+   For more general information and a description of RBD QoS configuration
+   options, refer to <xref linkend="rbd-qos" />.
+  </para>
+ </tip>
+ <para>
+  The QoS options can be configured at different levels.
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    Globally
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    On a per-pool basis
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    On a per-image basis
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  The <emphasis>global</emphasis> configuration is at the top of the list and
+  will be used for all newly created RBD images and for those images that do
+  not override these values on the pool or RBD image layer. An option value
+  specified globally can be overridden on a per-pool or per-image basis.
+  Options specified on a pool will be applied to all RBD images of that pool
+  unless overridden by a configuration option set on an image. Options
+  specified on an image will override options specified on a pool and will
+  override options specified globally.
+ </para>
+ <para>
+  This way it is possible to define defaults globally, adapt them for all RBD
+  images of a specific pool, and override the pool configuration for
+  individual RBD images.
+ </para>
+ <sect3 xml:id="dash-rbd-qos-global">
+  <title>Configuring Options Globally</title>
+  <para>
+   To configure the &rbd; options globally, select
+   <menuchoice><guimenu>Cluster</guimenu>
+   <guimenu>Configuration</guimenu></menuchoice> from the main menu.
+  </para>
+  <procedure>
+    <step>
       <para>
-       On the <emphasis>primary</emphasis> cluster's &dashboard;, create an RBD
-       image so that its parent pool is the pool that you already created for
-       replication purposes. Enable the <option>Exclusive lock</option> and
-       <option>Journaling</option> features for the image. Refer to
-       <xref linkend="dashboard-rbds-create"/> for details on how to create RBD
-       images.
+        To list all available global configuration options, next to
+        <menuchoice><guimenu>Level</guimenu>, choose
```

Comment:
", choose" must not be outside of the menuchoice element here. :)

Reply:
Thank you!!